CommunifyLibrary/NonPersistentModels/ParameterModels/ChangePasswordResponse.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace CommunifyLibrary.NonPersistentModels.ParameterModels
+{
+    public class ChangePasswordResponse
+    {
+        public string ReplyMessage { get; set; }
+
+        public bool IsSuccess { get; set; }
```

Comment:
Why? 200 is succes

Reply:
IsSuccess ile methodun başarılı olup olmadığını değil, change işleminin başarılı olup olmadığını kontrol ediyoruz.

Reply:
Gördim. bunun için zaten httpnin kendisi durum kodları vermiş. Bknz: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status

Reply:
halletik ama exception messageları frontendte yakalayamadık. O yüzden controller string data dönüyor. Bu data da yakaladığımız exceptionların mesajları.